src/app/rpmostree-builtin-upgrade.c

```diff
@@ -125,7 +125,8 @@ rpmostree_builtin_upgrade (int             argc,
                                        opt_allow_downgrade,
                                        FALSE,  /* skip-purge */
                                        FALSE,  /* no-pull-base */
-                                       FALSE); /* dry-run */
+                                       FALSE,  /* dry-run */
+                                       FALSE); /* no-overrides */
```

Comment:
Not necessary to redo in this patch, but at this point I'm wondering if it'd be saner to have the command lines construct GVariants directly or so.